subsys/net/lib/app/ieee802154_settings.c

```diff
@@ -58,5 +58,7 @@ int _net_app_ieee802154_setup(void)
 	}
```

Comment:
The commit message of patch "net/lib: net_app can put 15.4 interface up relevantly" sounds confusing, what about just "The 15.4 interface is set UP after the required channel is configured."

Reply:
better yes